keyboards/cannonkeys/instant60/info.json

```
@@ -0,0 +1,15 @@
+{
+    "keyboard_name": "Instant60",
+    "url": "https://cannonkeys.com",
+    "maintainer": "CannonKeys",
```

`maintainer` should actually be your GitHub username.

```suggestion
    "maintainer": "awkannan",
```